appstore/dev_portal_api.py

```diff
@@ -42,6 +42,90 @@ def me():
     })
 
 
+@legacy_api.route('/users/me/developer', methods=['GET'])
+def my_apps():
+    result = authed_request('GET', f"{config['REBBLE_AUTH_URL']}/api/v1/me/pebble/appstore")
+    if result.status_code != 200:
+        abort(401)
+    me = result.json()
+
+    developer_id = me['id']
+    # Get apps
+    my_apps = [x for x in App.query.filter_by(developer_id=developer_id)]
+    my_appdata = []
+    for a in my_apps:
+        my_appdata.append({
+            "id": a.id,
+            "title": a.title
+        })
```

Comment:
List comprehensions are cool!

```suggestion
    my_appdata = [{"id": a.id, "title": a.title} for a in my_apps]
```